src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,687 @@
+				List of predefined regional flags:
+				"time": the timePattern member of regional option is used
+				List of explicit characters, which should have escape \\ character in front of them:
+					C, &, a, A, ?, L, 9, 0, #, >, <, y, M, d, h, H, m, s, t, f.
+				List of time-flags when explicit time pattern is used:
+				"t": first character of string which represents AM/PM field
+				"tt": 2 characters of string which represents AM/PM field
+				"h": hours field in 12-hours format without leading zero
+				"hh": hours field in 12-hours format with leading zero
+				"H": hours field in 24-hours format without leading zero
+				"HH": hours field in 24-hours format with leading zero
+				"m": minutes field without leading zero
+				"mm": minutes field with leading zero
+				```
+					//Initialize
+					$(".selector").%%WidgetName%%({
+						timeDisplayFormat: "time"
+					});
+
+					//Get
+					var timeDisplayFormat = $(".selector").%%WidgetName%%("option", "timeDisplayFormat");
+
+					//Set
+					$(".selector").%%WidgetName%%("option", "timeDisplayFormat", "time");
+				```
+			*/
+			timeDisplayFormat: null,
+			/* type="bool" Gets/Sets if the editor should only allow values from the list of items. Matching is case-insensitive.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					isLimitedToListValues : true
+				});
+
+				//Get
+				var limited = $(".selector").%%WidgetName%%("option", "isLimitedToListValues");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "isLimitedToListValues", false);
+			```*/
+			isLimitedToListValues: true,
+			/* type="bool" Disables/Enables default notifications for basic validation scenarios built in the editors such as required list selection, value wrapping around or spin limits.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					suppressNotifications : true
+				});
+
+				//Get
+				var suppressNotifications = $(".selector").%%WidgetName%%("option", "suppressNotifications");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "suppressNotifications", true);
+			```
+			*/
+			suppressNotifications: false,
+			/* type="bool" Gets/Sets whether the onscreen keyboard (if available on device) should be shown when the dropdown button is clicked/tapped. This option prevents initial focus or removes it when the drop button is clicked/tapped.
+				Note: The option does not perform device detection so its behavior is always active if enabled.
+				Note: When drop down is opened the only way to close it will be using the drop down button.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					suppressKeyboard : true
+				});
+
+				//Get
+				var readOnly = $(".selector").%%WidgetName%%("option", "suppressKeyboard");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "suppressKeyboard", true);
+				```
+			*/
+			suppressKeyboard: false,

```

Comment:
The same as above comment

Reply:
This is ignored in the parent control, so it should be added